zerver/webhooks/gogs/doc.html

```diff
@@ -0,0 +1,41 @@
+
+<p>
+    First, create the stream you'd like to use for Gogs Webhook notifications,
```

Comment:
"use for Gogs webhook notifications, and then subscribe"

You might also want to add a link to the "Create a stream" user doc at "create the stream"